quiche/src/cid.rs

```diff
@@ -425,10 +425,6 @@ impl ConnectionIdentifiers {
     ///
     /// Returns an error if the provided Connection ID or its metadata are
     /// invalid.
-    ///
-    /// Returns a list of tuples (DCID sequence number, Path ID), containing the
-    /// sequence number of retired DCIDs that were linked to their respective
-    /// Path ID.
```

Comment:
This comment has been deleted but there was no code change. Is the comment actually wrong (and we can just fix that up in a quick standalone PR) or is there something else at hand?

Reply:
The comment was out of date. iirc the function was changed but the comment was never updated.

Reply:
In that case, please create a separate PR with the change